assets/app/styles/_core.less

```diff
@@ -648,6 +648,11 @@ label.checkbox {
   margin-left: 3px;
 }
 
+.image-summary-description {
```

Comment:
I was thinking like a .preserve-newlines class that does the word break and the pre wrap